subprojects/resources-http/src/test/groovy/org/gradle/internal/resource/transport/http/ApacheDirectoryListingParserTest.groovy

```diff
@@ -165,7 +165,7 @@ class ApacheDirectoryListingParserTest extends Specification {
         where:
         artifactRootURI                                                         | repoType
         "http://localhost:8081/artifactory/repo1/junit/junit/"                  | "artifactory"
-        "https://repo1.maven.org/maven2/junit/junit/"                           | "mavencentral"
+        "https://repo.maven.apache.org/maven2/junit/junit/"                           | "mavencentral"
```

Comment:
Can you align the pipe character with the other ones?

Reply:
Fixed. Sorry missed that.